### ISSUE TITLE
Complete Reference Manual

### ISSUE THREAD

**Issue by cmiya** [cwrc/HuViz#3]
- The reference manual should be on a separate webpage (the current drop down panel is too narrow include all of the necessary info, particularly if there are screenshots).
- The manual should include a table of contents that is always visible
- The table of contents should be organized into chapters. The chapter titles should reflect sample tasks. For example: loading a dataset, graphing nodes, labelling nodes, pinning nodes, shelving nodes, discarding vs hiding, retrieving nodes, adjusting graph settings.
- Sample tasks should be organized into numbered steps.
- Sample tasks should be accompanied by screenshots. The screenshots should be labelled with numbers according to the steps.
  (**For examples of different ways to design effective Reference Manuals see Regex and Visual Eyes)

**Comment by cmiya** [comment]
I've created an outline of the layout for the Quick Reference Guide and Complete Reference Manual in the CWRC Documentation Folder (https://drive.google.com/drive/u/0/folders/0B6A14cazy_eERmZPLTVzc050eUk). 

The plan is to:
1) Complete the first draft of the Quick Reference Guide (tables + labelled screenshot)
2) Make any final edits/changes (in consultation with Mihaela, Susan and the rest of the group - as this will involve issues of definition)
3) Upload it to the HuViz Browser.

*After the Quick Reference Guide is finished, the next goal will be to complete the rough draft of the complete Reference Manual in DITA. Both the Quick Reference Guide and Complete Reference Manual will need to be finished and uploaded before user testing this winter.

**Comment by cmiya** [comment]
First draft of the Quick Reference Guide has been uploaded to the CWRC Documentation Folder:
https://drive.google.com/drive/u/0/folders/0B6A14cazy_eERmZPLTVzc050eUk

This version of the Quick Reference Guide includes only the tables (will be accompanied by labelled screenshots in the future). Even with keeping the descriptions short the tables are fairly large. Seems too big to display in the HuViz Toolbool. Could link to the tables in the Help Tab, so that they open as a new window AND/OR include the tables as sub-sections of the larger Reference Manual.

Thoughts??

**Comment by antimony27** [comment]
@cmiya I'm leaving this ticket open so we can start thinking about any updates that need to happen to the user test (I think that we were going to use that as a reference manual?), and where we should put it so the public can easily find this information.

**Comment by cmiya** [comment]
Here's a link to the current documentation (http://cwrc.ca/HuViz_Documentation). The images and steps for the how-to manual were actually used for the user tests. I think as a result of the tests, we ended up changing and improving some of the wording. It needs to be updated to include the latest features, but I think it's a good start!

**Comment by cmiya** [comment]
The link to the documentation is actually posted on huviz.dev.nooron.com, but it's a bit hidden. If you click on the **Help Tab** (?), there's a link to **"Manual"** at the top-right of the pop-up Quick Start window.

In order to make the complete documentation more visible/accessible, we could try posting the links in the Help Tab:

- Quick Start (instead of automatically popping up, the user would click here)
- Complete Documentation (link to http://cwrc.ca/HuViz_Documentation)
- Learn More about HuViz (link to CWRC profile page)
- etc.

**Comment by SusanBrown** [comment]
Agreed. Let’s do this asap. And have these links appear on the top right in the actual panel rather than a separate popup, as Chelsea suggests.

I just created a profile page for HuViz in CWRC and put in some basic content — please use http://beta.cwrc.ca/huviz

This page is now a submenu of the Technical menu on the About page for CWRC (http://beta.cwrc.ca/about).

Chelsea, you could talk to Mihaela about whether we can have a menu that is one level down, i.e. specific to this page, and also how to get the Technical menu box linking back so that you go back to the Technical page when you click on it.




On Apr 7, 2018, at 5:55 PM, cmiya <notifications@github.com<mailto:notifications@github.com>> wrote:


The link to the documentation is actually posted on huviz.dev.nooron.com<http://huviz.dev.nooron.com>, but it's a bit hidden. If you click on the Help Tab (?), there's a link to "Manual" at the top-right of the pop-up Quick Start window.

In order to make the complete documentation more visible/accessible, we could try posting the links in the Help Tab:

  *   Quick Start (instead of automatically popping up, the user would click here)
  *   Complete Documentation (link to http://cwrc.ca/HuViz_Documentation)
  *   Learn More about HuViz (link to CWRC profile page)
  *   etc.
